packages/terra-action-footer/CHANGELOG.md

```diff
@@ -7,6 +7,9 @@
 * Changed
   * Updated WDIO screenshots due to webpack-config-terra sass upgrade.
 
+* Added
```

Comment:
Might be a merge issue this and other change logs but this should be moved to the Unreleased section.

Reply:
Done